client/extensions/woocommerce/app/product-categories/form.js

```diff
@@ -216,6 +260,34 @@ class ProductCategoryForm extends Component {
 								<FormLabel htmlFor="description">{ translate( 'Description' ) }</FormLabel>
 								{ this.renderTinyMCE() }
 							</FormFieldSet>
+							<FormFieldSet>
+								<FormLabel>
+									<FormCheckbox
+										checked={ this.state.isTopLevel }
+										onChange={ this.onTopLevelChange }
+									/>
+									<span>{ translate( 'Top level category' ) }</span>
+								</FormLabel>
+
+								{ ! isTopLevel && (
+									<div>
+										<FormLabel>{ translate( 'Select a parent category' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **100** times:
`translate( 'Parent Category' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-product-cat-parent-field)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).